docs/lib/auth/user-attributes.md

```diff
@@ -0,0 +1,7 @@
+---
+title: User Attributes
```

Comment:
We should be consistent in capitalization - either every significant word starts with a capital as in here or only the first word does as in "Register a user"